### PR TITLE
Add GCP fake credentials

### PR DESCRIPTION
**Pull request by fao89**
https://django-storages.readthedocs.io/en/latest/backends/gcloud.html#authentication

Required PR: https://github.com/pulp/pulpcore/pull/3481

Signed-off-by: Fabricio Aguiar <fabricio.aguiar@gmail.com>

### CODE REVIEW

File: templates/github/.github/workflows/scripts/install.sh.j2 [hunk 1]
```diff
@@ -204,9 +204,17 @@ fi
 {%- endif %}
 
 {%- if test_gcp %}
-
+cat >> credentials.json << GCP_JSON
+{
```
Comment (ipanova) [inline templates/github/.github/workflows/scripts/install.sh.j2:208]: just to understand this better, where this is taken from?

Comment (fao89) [reply templates/github/.github/workflows/scripts/install.sh.j2:208]: from:
https://cloud.google.com/sdk/docs/install
https://cloud.google.com/docs/authentication/provide-credentials-adc#local-dev

Comment (ipanova) [reply templates/github/.github/workflows/scripts/install.sh.j2:208]: I pulled in these changes into my pr and the issue seems to persist https://github.com/pulp/pulp_container/pull/1196

Comment (fao89) [reply templates/github/.github/workflows/scripts/install.sh.j2:208]: it seems it needs more hacks:
https://github.com/fsouza/fake-gcs-server#using-with-signed-urls

**Comment by ipanova** [comment]
@fao89 I don't think your changes are still enough, because pulp-container CI fails. Will you be able to test these changes on the example of pulp-container https://github.com/pulp/pulp_container/pull/1196

**Comment by ipanova** [comment]
@pulp/deployments if someone can take this over and finalize, it  would be great. We have already merged the corresponding PR in core that states we GCP is supported https://github.com/pulp/pulpcore/pull/3481 hovewer we do not have a proof of a successful flow because the CI part was not done.

**Comment by fao89** [comment]
actually, this PR is more related to @pulp/continuous-integration team

**Pull request closed without merging**